<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>612</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>731</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>2000</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/splash2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>710</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editUser">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>密 码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editPSWD">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="chkBoxSave">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>保存用户名</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>434</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="btnOK">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
      <property name="icon">
       <iconset resource="C:/Users/Lenovo/Desktop/资源/QT5.9Samp2019/chap06Forms/samp6_5SplashLogin/res.qrc">
        <normaloff>:/images/images/704.bmp</normaloff>:/images/images/704.bmp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="icon">
       <iconset resource="C:/Users/Lenovo/Desktop/资源/QT5.9Samp2019/chap06Forms/samp6_5SplashLogin/res.qrc">
        <normaloff>:/images/images/706.bmp</normaloff>:/images/images/706.bmp</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
  <include location="C:/Users/Lenovo/Desktop/资源/QT5.9Samp2019/chap06Forms/samp6_5SplashLogin/res.qrc"/>
 </resources>
 <connections/>
</ui>
